### DISCUSSION TITLE
WinEdt with Zotero

### DISCUSSION THREAD

**Discussion by Alnoghashi** [WinEdt-Team/WinEdt-Support#55] [1.1 General Help and Discussion]
Is there an integration between WinEdt-11 and Zotero?

**Comment by Alex-Simonic** [comment]
I don't use Zotero but if there are users that want this functionality in WinEdt and have Zotero and BBT plugin configured I could help with a WinEdt add-on (macro) to implement "CITE AS YOU WRITE" function for those that are comfortable with such server-based functionality. After all, it is just a matter of executing something like:

`curl -s http://127.0.0.1:23119/better-bibtex/cayw?format=pandoc^&brackets=true
`

I will not get involved with different formats and Zotero/BBT particulars. From what I saw on [CAYW](https://retorque.re/zotero-better-bibtex/citing/cayw/index.html) about the "integration" with other editors (vim, emacs,...) the above command is the key for this functionality (although security concerns are always legitimate with such server-based functions). It is up to you or other interested users to give me more details before I can give you more specific instructions.
